docs/resources/User.md

```diff
@@ -14,6 +14,21 @@ data:image/jpeg;base64,BASE64_ENCODED_JPEG_IMAGE_DATA
 
 Ensure you use the proper header type (`image/jpeg`, `image/png`, `image/gif`) that matches the image data being provided.
 
+## Usernames and Nicknames
+
+Limitations on Discord usernames are few and far between. We do, however, have a few of which developers should be mindful. Discord usernames and nicknames are restricted to a minimum of `2` unicode characters and a maximum of `32`.
```

Comment:
The first part of this sentence doesn't seem to provide any meat or value to the paragraph as a whole, and the rest has a lot of information jammed into a paragraph blurb. I would personally adjust this to use a array like we use elsewhere, e.g. something like:

1. Usernames can contain most valid unicode characters (we limit some zero-width or non rendering characters)
2. Usernames must be between 2 and 32 characters long
3. Usernames cannot contain '@' '#' ':' '\```'
4. Usernames cannot be "discordtag", "everyone", "here"